src/algebra/group/defs.lean

```diff
@@ -211,6 +211,21 @@ mul_one_class.mul_one
 
 attribute [ematch] add_zero zero_add -- TODO(Mario): Make to_additive transfer this
 
+end mul_one_class
```

Comment:
The changes to this file look like a bad merge

Reply:
Yeah, this was git's work! I've removed them locally and I'm just looking over the rest.